packages/NodeTypeResolver/TypeAnalyzer/ArrayTypeAnalyzer.php

```diff
@@ -87,6 +96,41 @@ private function isIntersectionArrayType(Type $nodeType): bool
         return true;
     }
 
+    private function isPropertyFetchWithArrayDocblockWithoutDefault(Node $node): bool
```

Comment:
```suggestion
    private function isPropertyFetchWithArrayDocblockWithoutDefault(PropertyFetch|StaticPropertyFetch $node): bool
```

Reply:
It's actually called from 2 places, where one is as you say, but the other takes always a `Node`:

PHPStan then reports:

![image](https://user-images.githubusercontent.com/924196/144762601-afa1e53f-fe55-433d-876b-3fd0b4819a98.png)